Sources/Concrete/Controller/SpeechController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  SpeechController.swift
+//  InstantSearch
+//
+//  Created by Robert Mogos on 18/12/2017.
+//
+
+import UIKit
+import Speech
+
+/// A controller object that manages the speech recognition to text
+/// `SpeechController` is using the Speech framework, so it can only be used with iOS 10+
```

Comment:
You can put these comments right above the class definition so that they appear to the devs